vms/event/views.py

```diff
@@ -120,6 +147,15 @@ def get_context_data(self, **kwargs):
         job_obj = get_jobs_by_event_id(self.kwargs['event_id'])
         context['job_list'] = job_obj.values_list('start_date',
                                                   'end_date').distinct()
+        context['country_list'] = Country.objects.all()
+        event_id = self.kwargs['event_id']
+        event =  get_event_by_id(event_id)
+        context['country'] = event.country
+        context['country_id'] = event.country.pk
+        context['state'] = event.state
+        context['state_id'] = event.state.pk
+        context['city'] = event.city
+        context['city_id'] = event.city.pk
```

Comment:
I don't know if in future we implement the reuse of id will affect this or not. can you please check it once.

Reply:
These ids have nothing to do with VMS , they are of cities light. I don't think that it wil affect